start the navigation page (#2065)
grommet/hpe-design-system: aries-site/src/pages/templates/navigation.mdx

import { ExampleImagePreview } from '../../components';
import { Box } from 'grommet';
import { CardGrid } from '../../components';
import { getPageDetails, getRelatedContent } from '../../utils';

export const navPatterns = [
  'Drill Down Navigation',
  'Side to Side Navigation',
  'Reverse Navigation',
  'Matrix Navigation',
];
export const navCards = navPatterns.map(pattern => getPageDetails(pattern));

> Navigation is the overall structure that provides users screen direction to complete a task logically and easily through a website or an application. 

Reviewer: Active tense.

```suggestion
Navigation is the overall structure providing users screen direction to complete a task logically and easily through a website or an application. 
```

> Part of creating a seamless appplication or interface is creating an obvious flow from one view/interaction to the next. Think of Navigation as a story map where you help guide your user through each process. 

Reviewer: ```suggestion
Part of creating a seamless application or interface is creating an obvious flow from one view or interaction to the next. Think of navigation as a story map where you help guide your user through each process. 
```

## Types of Navigation

There are 4 different types of navigational directions a user can maneuver in:

<CardGrid cards={navCards} />

## What makes navigation successful

Users should always be able to tell where they are on a page, application or product and should be predictable as to how they can get to the next step or destination. A task has a start and an end point so it is important to keep the functionality or task(s) in mind when creating products with common user experiences and expectations.

> **Follow the points below for a successful Navigation:**

Reviewer 2: ```suggestion
**Follow the points below for successful navigation:**
```

- Identify the problem.
- Understand what the user’s role is and common processes followed.

> To learn more on what makes a successful Navigation, check out the [Figma file](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9473).

Reviewer 2: ```suggestion
To learn more on what makes navigation successful, check out the [Figma file](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9473).
```

import { ExampleImagePreview } from '../../components';
import { Box } from 'grommet';
import { CardGrid } from '../../components';
import { getPageDetails, getRelatedContent } from '../../utils';

export const navPatterns = [
  'Drill Down Navigation',
  'Side to Side Navigation',
  'Reverse Navigation',
  'Matrix Navigation',
];
export const navCards = navPatterns.map(pattern => getPageDetails(pattern));

Navigation is the overall structure that provides users screen direction to complete a task logically and easily through a website or an application. 

Part of creating a seamless appplication or interface is creating an obvious flow from one view/interaction to the next. Think of Navigation as a story map where you help guide your user through each process. 

> Consider the following flow below where a user is navigated from the HPE.com home page to a Sign Up page to create an account to the view of their dashboard. 

Reviewer 2: This sentence feels out of place. I think we should remove it unless there is going to be a visual/storyline to follow regarding this example.

Author: agree Ill take it out for now

## Types of  Navigation

There are 4 different types of navigational directions a user can maneuver in:

> <CardGrid cards={navCards} />

Reviewer 3: <img width="719" alt="Screen Shot 2021-11-18 at 12 19 21 PM" src="https://user-images.githubusercontent.com/62117454/142491397-c5883c03-21af-46a6-9742-5e22793cc176.png">

Can we add background back color where the imagery is please. 

Can we also make "Drill Down Nav" imagery smaller so it's aligned with the others please :)

Reviewer 2: @vavalos5 in order for the SVGs to play nicely like that -- can you re-export them within the bounds of the "invisible container" like we've done in other card preview here? https://www.figma.com/file/IvYbGVlucoUMvUgbYFSG7a/Design-System-Card-Site?node-id=713%3A12

Reviewer: Just confirming whether or not we were able to get replacement SVGs?

> ## What makes a successful Navigation

Reviewer 2: Wondering if we should rephrase to "What makes successful navigation" or "What makes navigation successful"?

This makes it seem like "Navigation" is a single thing, when in fact it's a flow of things and components.

Reviewer 2: @vavalos5

Reviewer 3: I agree @taysea , good change!

Users should always be able to tell where they are on a page, application or product and should be predictable as to how they can get to the next step or destination. A task has a start and an end point so it is important to keep the functionality or task(s) in mind when creating products with common user experiences and expectations.

**Follow the points below for a successful Navigation:**
- Identify the problem
- Understand what the user’s role is and common processes followed

> - Who is this being intended for and determine the most common tasks

Reviewer 2: Since the other bullet points start with verbs, we should phrase this one to start with a verb as well. It seems to be the same as the second bullet point (unless I'm misunderstanding), so maybe we can remove the third bullet point all together?

Reviewer 2: @vavalos5

Reviewer 3: Agreed @taysea

> To learn more on what makes a successful Navigation, check out the Figma file.

Reviewer 3: Are we going to include the Figma link here where it says "check out the Figma file"?

Author: yes I can include that!
